phpBB/language/en/acp/permissions_phpbb.php

```diff
@@ -79,6 +79,7 @@
 	'ACL_U_SAVEDRAFTS'	=> 'Can save drafts',
 	'ACL_U_CHGCENSORS'	=> 'Can disable word censors',
 	'ACL_U_SIG'			=> 'Can use signature',
+	'ACL_U_EMOJI'		=> 'Can use Emoji in topic title',
```

Comment:
Maybe better with plural? Emojis?

Reply:
We already use `Emoji` elsewhere

![immagine](https://user-images.githubusercontent.com/480857/71563680-007deb00-2a94-11ea-8655-22ec778c1efa.png)